scripts/train_agents.py

```diff
@@ -187,14 +191,15 @@ def set_input(args, quick_test=False):
 if __name__ == '__main__':
     args = get_arguments()
     quick_test = False
+    supporter_run = False
```

Comment:
I think it would be easier to add a new learner type called such as Originaler such that it's reward is just the original reward structure:

```
class Originaler(Learner):
    def calculate_reward(self, p_idx, env_info, ratio, num_players):
        super().extract_reward(p_idx, env_info, ratio, num_players)
        return self.personal_reward
```

This way the code will be more intuitive and anytime we want to use it we can just change the `args.learner_type`. And then we won't have to add all these new variables such as `supporter_run`, `args.reward_magnifier` and, the `if statement` from line 157 to 163.

Reply:
I will add this type, remove supporter_run and also if statement there. In addition, I will set the default to be Originaler(Traditionalist, LegacyAgent), but still keep the argument of args.reward_magnifier so as to allow users(ME, haha I need this argument for some experiments) to customize the reward setting.

Reply:
Does this plan sound great to you?

Reply:
Yep sounds great thanks!